lib/ipmi_sensor.c

```diff
@@ -288,36 +288,36 @@ ipmi_sensor_print_fc_threshold(struct ipmi_intf *intf,
 		thresh_available = 0;
 
 	if (csv_output) {
-                printf("%s,", sr->s_id);
-        if (sr->s_reading_valid) {
-            if (sr->s_has_analog_value)
-                printf(",%.3f,%s,%s",
-                       sr->s_a_val, sr->s_a_units, thresh_status);
-            else
-                printf(",0x%x,%s,%s",
-                       sr->s_reading, sr->s_a_units, thresh_status);
-        } else {
-            printf(",%s,%s,%s",
-                   "na", sr->s_a_units, "na");
-        }
-        if (thresh_available && sr->full) {
-#define PTS(bit, dataidx) {                     \
-print_thresh_setting(sr->full, rsp->data[0] & (bit),        \
-    rsp->data[(dataidx)], ",", "%.3f", "0xx", "%s");    \
+		printf("%s", sr->s_id);
+		if (sr->s_reading_valid) {
+			if (sr->s_has_analog_value)
+				printf(",%.3f,%s,%s",
+				       sr->s_a_val, sr->s_a_units, thresh_status);
+			else
+				printf(",0x%x,%s,%s",
+				       sr->s_reading, sr->s_a_units, thresh_status);
+		} else {
+			printf(",%s,%s,%s",
+			       "na", sr->s_a_units, "na");
+		}
+		if (thresh_available && sr->full) {
+#define PTS(bit, dataidx) { \
+print_thresh_setting(sr->full, rsp->data[0] & (bit), \
+	rsp->data[(dataidx)], ",", "%.3f", "0x%x", "%s"); \
 }
-            PTS(LOWER_NON_RECOV_SPECIFIED,  3);
-            PTS(LOWER_CRIT_SPECIFIED,   2);
-            PTS(LOWER_NON_CRIT_SPECIFIED,   1);
-            PTS(UPPER_NON_CRIT_SPECIFIED,   4);
-            PTS(UPPER_CRIT_SPECIFIED,   5);
-            PTS(UPPER_NON_RECOV_SPECIFIED,  6);
+			PTS(LOWER_NON_RECOV_SPECIFIED, 3);
+			PTS(LOWER_CRIT_SPECIFIED, 2);
+			PTS(LOWER_NON_CRIT_SPECIFIED, 1);
+			PTS(UPPER_NON_CRIT_SPECIFIED, 4);
+			PTS(UPPER_CRIT_SPECIFIED, 5);
+			PTS(UPPER_NON_RECOV_SPECIFIED, 6);
 #undef PTS
-        } else {
-            printf
-                (",%s,%s,%s,%s,%s,%s",
-                 "na", "na", "na", "na", "na", "na");
-        }
-        printf("\n");
+		} else {
+			printf
+			     (",%s,%s,%s,%s,%s,%s",
+			     "na", "na", "na", "na", "na", "na");
```

Comment:
Please avoid breaking the line before the opening parenthesis. I know there are places in this very file where this has been done previously, but let's not introduce more of this.

I would appreciate if you broke this `ipmi_sensor_print_fc_threshold()` function with a separate commit into smaller pieces to make it shallower indented.

Reply:
First part has been fixed in 48fe375, but not quite sure about last part.
Firstly I am not the best C coder, but I could give a shot!

Do you mean something like this?
https://github.com/ipmitool/ipmitool/blob/c70665c3d1ae8d19634eb2ea386d1065a4cff506/lib/ipmi_user.c#L296-L302

Reply:
Yes, something like that.

Reply:
How is 6223ec1? Is that good enough?